contracts.json

```diff
@@ -93,19 +93,28 @@
   },
   "mumbai": {
     "BridgedPolygonNORI": {
-      "proxyAddress": "0x6E570A15C0C39b5F29388DEEa6ebda8CDDc40587"
+      "proxyAddress": "0x8D7713C76A2A6Ec517FB3C032524dd86425E24f4"
     },
     "Removal": {
-      "proxyAddress": "0xa484c59eB391afFd88C7de0dBF4B6E7e2A9E84fE"
+      "proxyAddress": "0xa051E9EeaC803d2fCA1DbF415b78AD9BfEB723b0"
     },
     "Certificate": {
-      "proxyAddress": "0xF5bafc8B26Fb38ea788Bc6eAf3d84ba3bc4Ca91C"
+      "proxyAddress": "0xe77534eF840B83b715994Bf339b04F81997b9477"
     },
     "Market": {
-      "proxyAddress": "0x61819E8238A3493547Ae1131dc657D87F62833FD"
+      "proxyAddress": "0x53D76F0d411F8Fc5278999Ae7929C163Dbeac7d7"
     },
     "LockedNORI": {
       "proxyAddress": "0xCdcB43CB7B668F0C1cA04FE4b60DA7F8C62Be393"
+    },
+    "RestrictedNORI": {
+      "proxyAddress": "0x0924196d683daBe242216Eb3e27bcf8FccCeCaD1"
+    },
+    "LockedNORIV2": {
```

Comment:
Hmm this is interesting. At this level I think it would have made more sense for this to replace `LockedNORI`.  Also it looks like this deploy replaced the `LockedNORI` proxy rather than upgrading it which means we've lost all the state.

Reply:
if it replaced the proxy, doesn't that mean the old proxy/implementation is still out there and we could still upgrade it?